common/src/main/java/xyz/wagyourtail/jsmacros/client/api/library/impl/FWorld.java

```diff
@@ -107,6 +116,19 @@ public List<PlayerListEntryHelper> getPlayers() {
         }
         return players;
     }
+
+    /**
+     * @param name the name of the player to get the entry for
+     * @return player entry for the given player's name or {@code null} if not found.
+     *
+     * @since 1.8.4
+     */
+    public PlayerListEntryHelper getPlayerEntry(String name) {
+        ClientPlayNetworkHandler handler = mc.getNetworkHandler();
+        assert handler != null;
+        PlayerListEntry entry = handler.getPlayerListEntry(name);
+        return entry != null ? new PlayerListEntryHelper(entry) : null;
```

Comment:
do we really need this function. doesn't the list one cover this since it should be easy enough on the js side...

and/or should this be uuid instead of name/another function for uuid

Reply:
You have to keep in mind that many users are not that experienced with JavaScript or coding in general. Many of the methods I have added in this PR could be easily done in JavaScript or any other scripting language for that matter, but are still nice to have nonetheless. And while in JavaScript you can do something like `Java.from(World.getPlayers()).find(e => e.getName() === "PlayerName");`, doing the same thing in lua would be much harder and in general produce a lot of  boilerplate code. Often you also want to get the entry of a specific player, so that's why I have added this method.
I would not replace this method with the UUID as most users are probably more familiar with normal names. Adding another method for the UUID is certainly an option.

Reply:
> doing the same thing in lua would be much harder and in general produce a lot of boilerplate code.

```lua
for _, v in ipairs(World.getPlayers()) do
  if v.getName() == "PlayerName" then
    -- doSomething
    break
  end
end
```

yeah, I guess.

> You have to keep in mind that many users are not that experienced with JavaScript or coding in general.

honestly, I don't concider this a valid reason

Reply:
So can I leave it in there?

Reply:
Yep